cprover_bindings/src/goto_program/expr.rs

```diff
@@ -553,6 +562,11 @@ impl Expr {
         Self::float_constant(c)
     }
 
+    /// `typ x[__CPROVER_infinity()] = >>> {elem} <<<`
```

Comment:
Can you please improve this doc a bit? I have no idea what this means.

Reply:
Done